Source/Modules/python.cxx

```diff
@@ -836,7 +834,7 @@ class PYTHON:public Language {
       Printf(default_import_code, tab8 tab4 "return importlib.import_module('%s')\n", module);
       Printf(default_import_code, tab4 "%s = swig_import_helper()\n", module);
       Printv(default_import_code, tab4, "del swig_import_helper\n", NULL);
-      Printv(default_import_code, "elif _swig_python_version_info >= (2, 6, 0):\n", NULL);
```

Comment:
I think we should raise a RuntimeError if < 2.6 saying unsupported version to give a helpful error message to users trying to use older versions.

Also the C code should `#error` if < 2.6.

Reply:
Ah, I just saw elsewhere you have already put in the check in the C code, so ignore that comment